CHANGELOG.md

```diff
@@ -24,8 +24,11 @@ docs/releases/Hotfix-Documentation-Best-Practices.md
 
 INTERNAL NOTE
 -->
+## Flutter 3.29 Changes
+
+### [3.29.2](https://github.com/flutter/flutter/releases/tag/3.29.2)
 
-## Flutter 3.29.1 Changes
+- [flutter/164036](https://github.com/flutter/flutter/pull/164036) - Increase the maximum number of concurrent background image decode tasks to 64.
```

Comment:
I dont think this matches our style guide for changelog entries. 
https://github.com/flutter/flutter/blob/master/docs/releases/Hotfix-Documentation-Best-Practices.md 
The suggestion is my attempt to match the changelog guidance based on reading the bug and pr. 
```suggestion
- [flutter/164036](https://github.com/flutter/flutter/pull/164036) - On iOS devices Increase number of concurrent background image decode tasks to partially mitigate "Image upload failed due to loss of GPU access" errors.
```